### PR TITLE
Add 'Ask GIS' panel and reorganize SCSS

### PR DESCRIPTION
**Pull request by jazairi**
The main purpose of this PR is to add 'Ask GIS' links to all views except the index, as specified in [GDT-129](https://mitlibraries.atlassian.net/browse/GDT-129). This is toggled by a feature flag so it does not appear in other TIMDEX UI apps. (However, a similar feature may be useful for those apps, too.)

Secondarily, this removes some rules from `timdexui.scss`. This is unrelated to the ticket, but I noticed a comment in that file suggesting that those rules should be relocated, and this seemed like a good opportunity to do so.  Check the commit message for details, as two rulesets were removed altogether.

#### Developer

- [x] All new ENV is documented in README
- [x] All new ENV has been added to Heroku Pipeline, Staging and Prod
- [x] ANDI or Wave has been run in accordance to
      [our guide](https://mitlibraries.github.io/guides/basics/a11y.html) and
      all issues introduced by these changes have been resolved or opened as new
      issues (link to those issues in the Pull Request details above)
- [ ] Stakeholder approval has been confirmed (or is not needed)

#### Code Reviewer

- [x] The commit message is clear and follows our guidelines
      (not just this pull request message)
- [ ] There are appropriate tests covering any new functionality
- [ ] The documentation has been updated or is unnecessary
- [x] The changes have been verified
- [x] New dependencies are appropriate or there were no changes

#### Requires database migrations?

NO

#### Includes new or updated dependencies?

NO

[GDT-129]: https://mitlibraries.atlassian.net/browse/GDT-129?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by jazairi** [comment]
Sanity check request for the reviewer: try running the tests locally and let me know if you see that weird behavior I shared in Slack. It seems no longer to be a problem for me with a fresh copy of the repo, for whatever reason.

File: app/views/record/_sidebar.html.erb [hunk 1]
```diff
@@ -8,11 +8,5 @@
     </div>
   <% end %>
 
-  <div class="bit askus">
```
Comment (JPrevost) [inline app/views/record/_sidebar.html.erb:11]: Aren't we losing this default contact logic the way this is implemented?

I wonder if we could continue down the path you started here and have a single `shared/ask` partial and within that partial split out the gdt/non-gdt logic?

**Comment by JPrevost** [comment]
@jazairi tests worked as expected for me locally

File: app/views/record/_sidebar.html.erb (hunk 1 above)
Comment (jazairi) [reply app/views/record/_sidebar.html.erb:11]: Yeah, I think that makes sense. The original 'Ask Us' bit felt kind of placeholder-y, but that's not my assumption to make.

**Comment by jazairi** [comment]
@JPrevost Sorry I missed your review on this. I just pushed a commit that I think addresses your feedback. To minimize complexity, I removed the `p` tag in the non-GDT ask us bit and normalized the titles for both. Personally, I prefer the more minimal design that Darcy proposed and figure it should apply to all TIMDEX UI apps, but I don't mind walking that back if you think changing non-GDT stuff is out of scope here.

File: app/views/record/_sidebar.html.erb [hunk 2]
```diff
@@ -8,11 +8,5 @@
...
-    <h3 class="title">Ask Us</h3>
-    <p>Not finding what you're looking for?  We're here to help with anything from quick questions to in-depth research.</p>
-    <p>
-      <a class="btn button-secondary" href="https://libraries.mit.edu/ask/">Ask Us</a>
-    </p>
-  </div>
+  <%= render 'shared/ask_gis' %>
```
Comment (JPrevost) [inline app/views/record/_sidebar.html.erb:11]: create-review-app

Comment (JPrevost) [inline app/views/record/_sidebar.html.erb:11]: Can we rename this partial to `shared/ask` now that it is handling both use cases?

Comment (jazairi) [reply app/views/record/_sidebar.html.erb:11]: D'oh. Yes.

**Review by JPrevost — APPROVED** [review]

**Pull request merged**